spec/unit/enforcement_spec.rb

```diff
@@ -1,15 +1,29 @@
 # frozen_string_literal: true
+class MyController # < ApplicationController
+  include Blacklight::AccessControls::Enforcement
+end
+
 describe Blacklight::AccessControls::Enforcement do
-  let(:controller) { CatalogController.new }
-  let(:search_builder) { SearchBuilder.new(method_chain, controller) }
-  let(:method_chain) { SearchBuilder.default_processor_chain }
+  let(:controller) do
+    c = MyController.new
+    allow(c).to receive(:current_ability).and_return(ability)
```

Comment:
@atz you can define the class within the describe block, and then include a method that returns ability.  Then use additional let statements to set that value. @jcoyne do you have an example?  I spent a little bit of time looking through sufia and cc and did not find one.